examples/player/simid_player.html

```diff
@@ -123,8 +123,10 @@
       let adParamsObj = '';
       switch (value) {
         case 'simid_overlay.html':
+          document.getElementById("linear_ad").click();
```

Comment:
It's really strange to use click here, but I guess it's fine.

A more common way to do this might be:
  setAdTypeLinear();

Then have a function
function setAdTypeLinear() {
  document.getElementById("linear_ad").checked;
  setInputsForAdType();
}

Either way seems fine though.

Reply:
I like this way better actually!